### PR TITLE
feat: enable return id flag for insert query

### PR DESCRIPTION
**Pull request by EnriqueL8**
Adding return id flag to insert query to be used in PostgreSQL and MySQL plugins.

### CODE REVIEW

File: Sources/SwiftKuery/Insert.swift [hunk 1]
```diff
@@ -36,14 +36,17 @@ public struct Insert: Query {
     /// An array of `AuxiliaryTable` which will be used in a query with a WITH clause.
     public private (set) var with: [AuxiliaryTable]?
     
+    /// A flag for return ID
+    public private (set) var returnID: Bool?
```
Comment (tunniclm) [inline Sources/SwiftKuery/Insert.swift]: I don't understand why this `Bool` is optional, aren't there only two options (true or false)?

Comment (EnriqueL8) [reply Sources/SwiftKuery/Insert.swift]: Yes, will change that

File: Sources/SwiftKuery/Insert.swift [hunk 2]
```diff
@@ -36,14 +36,17 @@ public struct Insert: Query {
     /// An array of `AuxiliaryTable` which will be used in a query with a WITH clause.
     public private (set) var with: [AuxiliaryTable]?
     
+    /// A flag for return ID
+    public private (set) var returnID: Bool
```
Comment (tunniclm) [inline Sources/SwiftKuery/Insert.swift]: Is there something stopping this being `public let`?

File: Sources/SwiftKuery/QueryError.swift [hunk 3]
```diff
@@ -30,6 +30,8 @@ public enum QueryError: Error {
     case unsupported(String)
     /// An error in transaction.
     case transactionError(String)
+    /// Invalid table definition
+    case invalidTableDefinition(String)
```
Comment (tunniclm) [inline Sources/SwiftKuery/QueryError.swift]: NOTE: This is a breaking change to the API

File: Sources/SwiftKuery/Insert.swift (hunk 2 above)
Comment (EnriqueL8) [reply Sources/SwiftKuery/Insert.swift]: No, will change that

File: Sources/SwiftKuery/QueryError.swift (hunk 3 above)
Comment (EnriqueL8) [reply Sources/SwiftKuery/QueryError.swift]: Realised this is not necessary

**Comment by EnriqueL8** [comment]
Test Passed ready to merge

**Pull request merged**